src/js/popup.js

```diff
@@ -1074,9 +1078,20 @@ Logic.registerPanel(P_CONTAINER_DELETE, {
   prepare() {
     const identity = Logic.currentIdentity();
 
-    // Populating the panel: name and icon
+    // Populating the panel: name, icon, and warning message
     document.getElementById("delete-container-name").textContent = identity.name;
 
+    const totalNumberOfTabs = identity.numberOfHiddenTabs + identity.numberOfOpenTabs;
```

Comment:
Whoops, I approved too early. An optional refinement to eliminate a little bit of duplicate functionality ...
```
     let warningMessage = "";
     if (totalNumberOfTabs > 0) {
      const grammaticalNumTabs = totalNumberOfTabs > 1 ? "tabs" : "tab";
      warningMessage = `If you remove this container now, ${totalNumberOfTabs} container ${grammaticalNumTabs} will be closed.`;
    }
    document.getElementById("delete-container-tab-warning").textContent = warningMessage;
```

Reply:
@groovecoder Hey! sorry for the delay, this surely makes the logic concise. Should I make these changes and push?

Reply:
Yes pelase. 👍

Reply:
Pushed the changes, but the Travis build seems to be failing somehow 😭